owasm/src/ext/mod.rs

```diff
@@ -29,3 +40,80 @@ where
     // TODO
     oei::get_external_data(external_id, 0).unwrap().parse::<T>().ok().unwrap()
 }
+
+pub fn load_median<T>(external_id: i64) -> Option<T>
+where
+    T: std::str::FromStr + std::cmp::PartialOrd + std::marker::Copy + num::Num + num::NumCast,
+{
+    let mut vals: Vec<T> = Vec::new();
```

Comment:
Let's have a separate function to do this loading for both `load_average` and `load_median`